.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches: ["ros2"]
+  pull_request:
+    branches: ["*"]
```

Comment:
Seems like this would run for PRs on the ros1 branch as well, right? That's why it was previously set to ros2

Reply:
I'm pretty certain that the action yml from the PR branch is what github runs, not the yml from the default or target branch. So we just need to put the appropriate configuration in ros1 and ros2 branches and not worry about this branches wildcard.